web/src/main/kotlin/com/example/ajaxproject/service/UserServiceImpl.kt

```diff
@@ -3,24 +3,24 @@ package com.example.ajaxproject.service
 import com.example.ajaxproject.dto.request.UserDTO
 import com.example.ajaxproject.exeption.NotFoundException
 import com.example.ajaxproject.model.User
+import com.example.ajaxproject.model.toUserDTO
 import com.example.ajaxproject.repository.UserRepository
 import com.example.ajaxproject.service.interfaces.UserService
-import org.bson.types.ObjectId
 import org.springframework.stereotype.Service
 
 @Service
 class UserServiceImpl(
     private val userRepository: UserRepository
 ) : UserService {
 
-    override fun createUser(userDTO: UserDTO): User {
+    override fun createUser(userDTO: UserDTO): UserDTO {
         val user = toEntity(userDTO)
-        return userRepository.save(user)
+        return userRepository.save(user).toUserDTO()
     }
 
     fun toEntity(userDTO: UserDTO): User {
         return User(
-            id = ObjectId().toHexString(),
+            id = userDTO.id,
             email = userDTO.email,
             password = userDTO.password,
         )
```

Comment:
Maybe better rewrite this function to extension function?

```suggestion
        fun UserDTO.toEntity() = User(
        id = ObjectId().toHexString(),
        id = id,
        email = email,
        password = password,
    )
```

Reply:
I will think about it. In fact i need to refactor all uses of UserDtTO to some UserRequest and UserResponse  and then it will be changed